gradle/versions.gradle

```diff
@@ -68,7 +68,7 @@ ext {
     javaAssistVersion = "3.27.0-GA"
     yubikitAndroidVersion = "2.3.0"
     yubikitPivVersion = "2.3.0"
-    powerliftAndroidVersion="1.0.0"
+    powerliftAndroidVersion="1.0.3"
```

Comment:
also something to cross check with partner teams.

Reply:
This is the change in the payload for powerlift https://office.visualstudio.com/PowerLift/_git/powerlift-android-sdk?path=/MIGRATION_GUIDE.md&version=GBmaster&_a=preview
This does not have any breaking change in the API that Auth app, LTW and CP are using. 
I also generated CP and Auth app apks and checked if we are able to upload an incident and it is working as expected. I will also keep the partner teams informed about this change

Reply:
Just curious, if you build AuthApp / CP, then which version is resolved through dependency resolution?

Reply:
It is 1.0.3, it resolves to the dependency we have adedd